code/modules/research/bepis.dm

```diff
@@ -0,0 +1,225 @@
+//This system is designed to act as an in-between for cargo and science, and the first major money sink in the game outside of just buying things from cargo (As of 10/9/19, anyway).
+
+//economics defined values, subject to change should anything be too high or low in practice.
+
+
+/obj/machinery/rnd/bepis
+	name = "\improper B.E.P.I.S. Chamber"
+	desc = "A high fidelity testing device which unlocks the secrets of the known universe using the two most powerful substances available to man: excessive amounts of electricity and capital."
+	icon = 'icons/obj/machines/bepis.dmi'
+	icon_state = "chamber"
+	layer = ABOVE_MOB_LAYER
+	density = TRUE
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 1500
```

Comment:
Leave this at 0 (from parent), add active_power_usage = 1500 and swap between use_power = IDLE_POWER_USE and ACTIVE_POWER_USE instead of reassinging this.